### ISSUE TITLE
Training Detectnet on COCO Dataset

### ISSUE THREAD

**Issue by benlemond** [NVIDIA/DIGITS#2112]
I am trying to train a detectnet model on the COCO dataset. I have downloaded the entire dataset, and I have run the coco2kitti.py script to generate the label files from the COCO annotations. I have made a subset of the data about ~6,000 images for training and ~1000 images for validation.

I performed the rest of the steps per a couple of tutorials, yet my training does not seem to be yielding any results at all. In fact, the charts in Digits only show "loss_bbox(train)" and "loss_coverage(train)". There is no mAP line, precision line, or any other line on my charts. Has anyone had these issues or successfully trained detectnet on the COCO dataset? Is there something I am missing in data preparation?

I loaded the dataset up to Digits
- Padding: 640x640
- Minimum box size: 25
- custom classes: dontcare,person
- rest = defaults

I created an object detection model
- epochs: 100
- subtract mean: none
- Solver: adam
- learning rate: 2.5e-05
- Policy: exponential decay
- Custom Network: https://github.com/dusty-nv/jetson-inference/blob/master/data/networks/detectnet.prototxt
- pretrained: bvlc_googlenet.caffemodel

**Comment by benlemond** [comment]
Here is the prototxt I am using
[detectnet_prototxt.txt](https://github.com/NVIDIA/DIGITS/files/2389783/detectnet_prototxt.txt)

**Comment by yeyaguang2018** [comment]
hi,That's the same with me. Will you solve it now?

**Comment by liuchang138929** [comment]
Did you fix it?  
I follow the guide to train the DetectNet in https://github.com/dusty-nv/jetson-inference 
same problem,,,

**Comment by jogiji** [comment]
Same issue here tooo

**Comment by jogiji** [comment]
Update : I tried with larger & different dataset and different learning rate than what was mentioned in tutorial and my model was successful in detecting the objects.
I wish detect net could be trained in tensorflow

**Comment by lt1103725556** [comment]
> Update : I tried with larger & different dataset and different learning rate than what was mentioned in tutorial and my model was successful in detecting the objects.
> I wish detect net could be trained in tensorflow

Do you get mAP curve finally?
Does it mean that when you use a small dataset you can't get the mAP curve?
